app/controllers/candidate_interface/restructured_work_history/break_controller.rb

```diff
@@ -8,6 +8,8 @@ def create
       @work_break = RestructuredWorkHistory::WorkHistoryBreakForm.new(work_history_break_params)
 
       if @work_break.save(current_application)
+        current_application.update!(work_history_completed: false)
```

Comment:
These work for me. Should they be covered in the system spec? (optional - not sure myself)

Reply:
Yea I'm never really sure about this one. I'll add it though.